doc/releases/release-notes-3.3.rst

```diff
@@ -23,6 +23,10 @@ Changes in this release
 Removed APIs in this release
 ============================
 
+* Removed Kconfig option ``CONFIG_USB_CDC_ACM_RINGBUF_SIZE`` in favor of
+  devicetree properties tx-fifo-size and rx-fifo-size, which
+  allow us finer configuration of CDC ACM UART per instance.
```

Comment:
Please add a reference to the compatible to the notes, so users know what you are talking about. This text is not very clear.

You can use this:

```
:dtcompatible:`zephyr,cdc-acm-uart`
```

to create a link to the compatible page in the bindings index  (https://docs.zephyrproject.org/latest/build/dts/api/bindings.html)